README.md

```diff
@@ -25,23 +25,53 @@ Currently tested platforms:
 
 #### On a Linux machine make sure to add your computer user to the ``docker`` group
 
+If your user is not added in the group, it causes a permission error when starting Docker.
+
+The instruction is [here](https://docs.docker.com/engine/install/linux-postinstall/).
+
+#### Make sure `docker swarm` works
+
+If you find the following error, please refer [this instruction](https://forums.docker.com/t/28428/8) to remove `--live-restore` option.
+
+```
+$ docker swarm init
+Error response from daemon: --live-restore daemon configuration is incompatible with swarm mode
+```
+
+Then make sure `docker swarm` works.
+
+```
+docker swarm init
+docker swarm leave --force
+```
+
+#### Make sure to enable cgroups v1.
+
+If you find the following error, plesae enable cgroups v1. The instraction is [here](https://fedoramagazine.org/docker-and-fedora-32/).
+
+```
+$ ./make.sh
+...
+ERROR: for chris_ultron_backend_queue_1  Cannot start service queue: OCI runtime create failed: this version of runc doesn't work on cgroups v2: unknown
+...
+```
 
 ### TL;DR
 
-If you read nothing else on this page, and just want to get an instance of the ChRIS backend services up and 
-running with no mess, no fuss:
+If you read nothing else on this page, and just want to get an instance of the ChRIS backend services up and running with no mess, no fuss:
 
 ```bash
 git clone https://github.com/FNNDSC/ChRIS_ultron_backend
 cd ChRIS_ultron_backend
 # Run full CUBE instantiation with tests:
-*unmake* ; sudo rm -fr FS; rm -fr FS; *make*
+./unmake.sh ; sudo rm -fr FS; rm -fr FS; ./make.sh
```

Comment:
I agree with these changes, `*make*` syntax is outdated copied-and-pasted stuff.